Polyhedron/demo/Polyhedron/Polyhedron_demo.cpp

```diff
@@ -64,7 +64,10 @@ Polyhedron_demo::Polyhedron_demo(int& argc, char **argv,
 
   mainWindow.setWindowTitle(main_window_title);
   mainWindow.show();
-
+#ifdef __APPLE__
+    system("osascript -e 'tell application \"System Events\" "
+      "to keystroke tab using {command down, shift down}'");
+#endif
```

Comment:
You cannot include that ugly hack in the code without a strong comments/documentation explaining why that is needed!